_projects/03-efficient-starai.md

```diff
@@ -39,11 +39,17 @@ excerpt: "Efficient Starai excerpt"
 ## 3. Lifted Continuous Inference
 
 ### People Involved:
+* Yuqiao Chen
 
 ### About:
+Continuous values are common in the real world, but in traditional graphical model, inference with continuous variable may requires to compute integal of potential functions without tractable close form. Because of this, lots of works about inference in continuous domain limits the potential to some special exponential family, which cannot fully represent the complexity of real world. Some other works focus on using sampling based method for inference. These method are powerful in the domain with both continuous and discrete values (Hybrid Model), but they are too computation-expensive to be applied to real word application when large number of variables are involved.
```

Comment:
Hey Yuqiao, I found a couple spelling/grammar enhancements:

* "... but in traditional graphical model**s**, inference with continuous variable**s** may requir**e computing integrals** of potential functions without **a** tractable close**d** form."
* "... lots of work on inference in continuous domain**s** limi**t** the potential..."
* "... complexity of **the** real world."
* "... sampling**-based methods**. These method**s**..."
* "... applied to **real-world applications** when **a** large..."